### DISCUSSION TITLE
Questions for GSoC/Outreachy coffee chat

### DISCUSSION THREAD

**Discussion by mloning** [alan-turing-institute/sktime#781] [Q&A]
Please feel free to add your questions that you would like us to discuss during the coffee chat on Thursday, 2:30pm BST.

**Comment by thayeylolu** [comment]
1. I would like to ask if Outreachy applicants can apply for the sktime mentorship program, and If we would be communicating with the mentors during our contribution stage.
2.  I would also like to know what factors determines the release of the next version of sktime. When are new features just enough to release a version.

**Comment by shreshtha48** [comment]
1) I am a Gsoc applicant. My project of interest is classification of EEG signals. I was looking over some techniques for analysis and there are some added features that I think could increase the accuracy of the project. Do I list them down as challenges? Or should I put them in the extended goals section?

2) I have little experience with python. I'm currently learning python. However, I am familiar with R and Matlab. I was wondering if this would hurt my chances of selection?

**Comment by Riyabelle25** [comment]
My GSoC proposal draft involves my implementing Forecasting methods for intermittent demand [(my ongoing work, # 730)](https://github.com/alan-turing-institute/sktime/pull/730) and adding new Composite methods (ensemblers) from the [Forecasting Wishlist.](https://github.com/alan-turing-institute/sktime/issues/220) 
I tried my best to find out more about the proposed Composition methods (from docs/FPP book) but couldn't. 
It would be nice if you could discuss what you guys had in mind while proposing these 3 in particular:
* `ColumnEnsembler`, 
* `FeatureUnion`, 
* a composite ensembler that ensemble over steps in forecasting horizon [I've named it the `StepEnsembler` :)]

Also, I'd like to know if implementing these are significant enough for a GSoC proposal, or are there other, more urgent Forecasting methods (from the Wishlist, or elsewhere) that need to be implemented- in which case I'd gladly take them up instead.

**Comment by fkiraly** [comment]
Apr 2021 has passed, hence this discussion is now obsolete..
